packages/shared/src/components/checklist/SquadEditWelcomePostChecklistStep.tsx

```diff
@@ -13,27 +11,24 @@ const SquadEditWelcomePostChecklistStep = ({
   squad,
   ...props
 }: ChecklistStepProps & { squad: Squad }): ReactElement => {
-  const { openModal } = useLazyModal();
   const welcomePostId = useFindSquadWelcomePost(squad)?.id;
   const { post: welcomePost } = usePostById({
     id: welcomePostId,
   });
 
   return (
     <ChecklistStep {...props}>
-      <Button
-        className="btn-primary"
-        disabled={!welcomePost}
-        icon={<EditIcon />}
-        onClick={() => {
-          openModal({
-            type: LazyModal.EditWelcomePost,
-            props: { post: welcomePost },
-          });
-        }}
-      >
-        Customize
-      </Button>
+      <div className="flex">
```

Comment:
Is this needed though?

Reply:
Yeah, for button to not wrap full width.